release-info/News

```diff
@@ -1,3 +1,59 @@
+OCaml 4.14.0 (?? March 2022)
+----------------------------
+
+- Integrated support for "go to definitions" in Merlin
+- Improved error messages in particular for module-level error
+- Standard library: new modules In_channel and Out_channel,
+  many new functions in Seq module, improved utf encoding support.
```

Comment:
I agree with @gasche that this is a confusing way to put it. We had encoding support for quite some time :–) 
I suggest `, string and bytes UTF decoding and validation support.`

Reply:
I would propose
```
- Standard library: new modules In_channel and Out_channel,
  many new functions in Seq module, UTF decoding and validation support
  for strings and bytes.
```
to reduce the size of the prefixed part of the nominal group.